### ISSUE TITLE
Small patches?

### ISSUE THREAD

**Issue by polux** [z0w0/helm#112]
Does 3564224d1ec60b31c091001620623fe00dbc4e42 mean that you don't accept small pull requests anymore or are these still Ok?

I'm sad to see this library go unmaintained, it was very promising. Thank you for all the good work!

**Comment by z0w0** [comment]
Hey mate. I'm sad to see it too, but I simply don't have the time unfortunately.
On top of this, I've started writing my hobby games in Rust - hence the little time I get isn't spent on Helm. Don't get me wrong, I truly believe in Haskell game dev, but I've found that Rust is a better option (for me personally). I think something similar to Helm in the future could really revolutionise game dev because I can see the power in the functional reactive paradigm.

I added that notice in the hopes that someone in the Haskell community will offer to maintain it. My hope is that someone can fork it (under the same name preferably) and I'll give them permission on Hackage so that they can release changes from their forked repo, and then this repo would remain here incase the forked repo goes chaotic.
